Makefile

```diff
@@ -79,6 +88,21 @@ validate: $(BUF)
 	go build -o bin/protoc-gen-go-cybozu-validate ./cmd/protoc-gen-go-cybozu-validate
 	$(RUN_BUF) generate --template buf.go-cybozu-validate.gen.yaml
 
+.PHONY: es
+es: $(BUF) $(PROTOC_GEN_ES)
+	cd $(ES_PACKAGES) && npm ci
+	$(RUN_BUF) generate --template ./buf.es.gen.yaml
+	cd $(ES_PACKAGES_PROTOBUF) && npm run build
+
+.PHONY: es-validate
+es-validate: es
```

Comment:
I tried using the remote plugin instead of the local plugin. does it work in your local machine? [d0360fe](https://github.com/cybozu/protobuf/pull/23/commits/d0360fe9912ed88fda4d925867e6589104e0993f)

Reply:
Thank you! It works!